corehq/apps/mocha/templates/mocha/base.html

```diff
@@ -64,4 +64,7 @@
       }
     </script>
   </body>
+  {% if show_status_page %}
```

Comment:
Do we need to show the status page on tests too? 
I believe we use mocha for running Frontend tests. 
@biyeun do you think it is required?

Reply:
yea i also left a comment about this. no we shouldn't include this on tests